.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+  schedule:
+    # Run everyday at 03:53 UTC
+    - cron: 53 3 * * *
+
+jobs:
+  test:
+    name: ${{ matrix.os }} / ${{ matrix.python-version }} / ${{ matrix.pip-version }}
```

Comment:
```suggestion
    if: >-
      !startsWith(matrix.python-version, 'pypy') ||
      github.event_name != 'pull_request'
```

Reply:
Hm... In that case `test` job is not run.

Reply:
Why? It should only exclude matching jobs.

Reply:
I may have made a mistake in the condition, though.

Reply:
No, it looks correct

Reply:
Should the condition be wrapped with `${{ ... }}`?

Reply:
No, that's only needed where you interpolate values.

Reply:
https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#about-contexts-and-expressions:
> When you use expressions in an `if` conditional, you may omit the expression syntax (`${{ }}`) because GitHub automatically evaluates the `if` conditional as an expression.

Reply:
Looks like the `matrix` parameters can't be used inside `if` at a job level:

![Screenshot 2020-05-05 at 04 26 18](https://user-images.githubusercontent.com/7377671/81015275-9c779300-8e88-11ea-9484-37faa403a75b.png)

Reply:
See also https://github.community/t5/GitHub-Actions/Matrix-cannot-be-used-in-jobs-level-if/td-p/41048

Reply:
That's because you have to apply my suggestion exactly like it's submitted. But you added a leading `!` there w/o quoting. This works only if you wrap the whole value with quotes or use a multiline form as I originally suggested.

Reply:
It's not a context access problem but a YAML parsing one.

Reply:
Ah, I see. Maybe you're right but that post is old, have you checked if it works with the proper syntax?

Reply:
![Screenshot 2020-05-05 at 04 35 23](https://user-images.githubusercontent.com/7377671/81016051-f167d900-8e89-11ea-9dcb-f724fb2ffea4.png)

Nope, doesn't work.

Reply:
:(